src/js/core/widget/core/Spin.js

```diff
@@ -189,6 +189,7 @@
 					item.style.transform = "translateY(-1000px)"; // move item from active area
 				}
 				item.style.opacity = change.opacity;
+				item.style.height = itemHeight + "px";
```

Comment:
This code is not available in wearable. What is the effect on the item?

Reply:
I checked in wearable profile.
It does not set the their height if useer change the itemHeight value.
So I added it.